examples/flows/chat/chat-basic-code-first/.env.example

```diff
@@ -0,0 +1,3 @@
+AZURE_OPENAI_API_KEY=<your_AOAI_key>
+AZURE_OPENAI_API_BASE=<your_AOAI_endpoint>
+AZURE_OPENAI_API_TYPE=azure
```

Comment:
https://github.com/microsoft/promptflow/pull/2030/files 

@D-W- please help to update the env variable name according to this PR>